x86_energy_plugin.cpp

```diff
@@ -122,18 +83,18 @@ class x86_energy_metric
         // needs move and default constructor though!
         x86_energy_metric(x86_energy_metric&&) = default;
         x86_energy_metric(const x86_energy_metric&) = delete;
-        x86_energy_metric& operator=(const x86_energy_metric&) = delete;
+        x86_energy_metric& operator=(const x86_energy_metric&) = default;
```

Comment:
Still no move assignment and copy-assign suddenly default instead of delete.

`x86_energy_metric& operator=(x86_energy_metric&&) = default;`